src/Views/OperatingSystemView.vala

```diff
@@ -129,13 +155,14 @@ public class About.OperatingSystemView : Gtk.Grid {
         button_grid.set_child_secondary (settings_restore_button, true);
 
         var software_grid = new Gtk.Grid () {
-            column_spacing = 12,
+            // The avatar has some built-in margin for shadows
+            column_spacing = 48 - 6,
```

Comment:
You'd mentioned wanting to drop this even more to 32px because of the illusion with the round image. I do think it would look better. 

```suggestion
            column_spacing = 48 - 6,
```